restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCDevice.java

```diff
@@ -370,6 +370,10 @@ public RCConnection connect(Map<String, Object> parameters, RCConnectionListener
             connection.state = RCConnection.ConnectionState.PENDING;
             DeviceImpl.GetInstance().sipuaConnectionListener = connection;
 
+            if (enableVideo == null) {
```

Comment:
@mehturt: have you tested that? https://docs.oracle.com/javase/7/docs/api/java/util/HashMap.html#get(java.lang.Object) says it returns null if key is not found, which afterwards is casted to Boolean in the code above which should make it false no?